src/view/com/util/post-ctrls/PostCtrls.tsx

```diff
@@ -141,105 +145,108 @@ let PostCtrls = ({
     shareUrl(url)
   }, [post.uri, post.author])
 
+  const btnStyle = React.useCallback(
+    ({pressed, hovered}: PressableStateCallbackType) => [
+      a.gap_xs,
+      a.rounded_full,
+      a.flex_row,
+      a.align_center,
+      a.justify_center,
+      {padding: 5},
+      (pressed || hovered) && t.atoms.bg_contrast_50,
```

Comment:
50 feels kinda dark to me, 25 is kinda our default bg state elsewhere

Reply:
I added a `hoverStyle` prop to `Button` to achieve this because `secondary`+`ghost` uses 100, is this ok? Alternatively, I could change `secondary`+`ghost` to be 25, or I could use a raw `<Pressable>` (not ideal)